File: FurnitureAPI/FurnitureAPI/Models/Review.cs
﻿using System;

namespace FurnitureAPI.Models
{
    public class Review
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string UserName { get; set; }
        public string FurnitureName { get; set; }
        public int FurnitureId { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime Date { get; set; }
    }
}

File: FurnitureAPP/app/src/main/java/com/example/furnitureapp/models/Review.kt
package com.example.furnitureapp.models

import com.google.gson.annotations.SerializedName

data class Review(
    val id: Int,
    val userId: Int,
    val userName: String,
    val furnitureName : String,
    val furnitureId: Int,
    val rating: Int,
    val comment: String?,
    val date: String
)